### ISSUE TITLE
ALT Text field  for images also in "Media Library" tab.

### ISSUE THREAD

**Issue by mark88008** [soflyy/oxygen-bugs-and-features#3325]
I just see the "ALT Text input field in the "URL" Tab for an image. It also should be in the "Media Library" Tab, because I never use the URL Tab.
